### ISSUE TITLE
Public View or Gust autologin

### ISSUE THREAD

**Issue by sanbroz** [advplyr/audiobookshelf#1135]
### Describe the feature/enhancement

Add a public view user for listening to audiobooks without login.

**Comment by advplyr** [comment]
Are you asking for the ability to disable authentication? Please expand on what you are looking for.

**Comment by sanbroz** [comment]
Yes, It will be nice if users can explore the catalog publicly and listen to books after login. We can also add a option to listen complete audiobook without login. A preview audio track can be added for public view. I am planning to use the project for running a community audio book website

**Comment by hackmonker** [comment]
Any future plans for this? interested in this too. Like I want to let the people listen to audibooks and search them without login but no admin features currently

**Comment by hobesman** [comment]
Interesting concept. Maybe add an option to make the account view only (no listening), sample only (30 seconds, or two minutes, or whatever), or full listening ability.

Most "users" are designed to track progress, and only envision one stream at a time. Those would have to be reconsidered for a guest account.

Would the guest account just not track/store progress? Or how would that be handled?

I believe someone else already requested a "preview" feature which could be made available to all users, and that progress obviously shouldn't be stored. 

But if you give the guest account full listening ability, I suspect that shouldn't be stored either, if there's a chance multiple people will use the same "guest" access. 

Basically any progress sync the server receives from the "guest" account could be responded to as normal (so the client can remain basically unmodified) but the server then simply discards the progress instead of saving it.

**Comment by hackmonker** [comment]
Yeah exactly. The guests can maybe listen to the book but wont store their progress. if they need to store progress, they must sign up

**Comment by Natetronn** [comment]
I'd like to see "Guest" libraries. Where we can create a new library, perhaps with a "Guests" toggle option; this so we can then add an item or multiple items to said library by doing a right click on an item, then "share to X" guest library (or something like that), which then adds the item to the shared guest library or libraries of our choosing. 

Then we can add users to said guest libraries like normal or turn off authentication/users completely for a specific library, as others have requested. Perhaps it would make use of "symbolic links" to books, this way books aren't duplicated and no real extra space is required.

**Comment by buddyhollyy** [comment]
I would also like an option to disable authorization on a local network. Not as interested in having 'guest' logins or libraries, but I could make that work...

**Comment by zaren-s** [comment]
I second @buddyhollyy - i run this locally and it would be so great to not have everyone have to remember their password. I would still want to have separate accounts for everyone to track their own progress - not just one guest account

In fact, I think the password policy is backwards - root user can be without password by regular users need one? It should be the other way around imo

**Comment by magicboy2000** [comment]
Hello, I am the requestor of #4291, from where I can get the version with feature of autologin?

**Comment by ptomli** [comment]
I'd like to have easy access for users, without needing a password. My instance is only for people at home, and neither my elderly mother nor my young daughter, need to "secure" their listening from prying eyes.

Jellyfin has a nice easy "who are you" mechanism, no password needed. I'd like something similar

<img width="665" height="301" alt="Image" src="https://github.com/user-attachments/assets/d97eeb04-8bd5-478c-b62e-51acfcbe566e" />
